### PR TITLE
Missing no-refresh for exos_facts in Interfaces

### PR DESCRIPTION
**Pull request by plazukin**
##### SUMMARY
Without the change exos_facts return timeout error when we use connection type network_cli. 
It happened because we send command without 'no-refresh' and script cli2json.py stuck in loop while reading console output

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
exos: exos_facts

##### ADDITIONAL INFORMATION
You can reproduction of the problem with this task and 'ansible_connection: network_cli':
```
    - name: Gather all facts
      exos_facts:
        gather_subset:
          - config
          - interfaces
```
I use the component with EXOS 16.2.5.4, but also face a problem in EXOS 30.5.1.15

### CODE REVIEW

**Comment by Andersson007** [comment]
@plazukin hi, thanks for the PR!
- we should ensure this change is not breaking (for older versions of the underlying technology)
- could you please add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) ?

cc @LindsayHill @ujwalkomarla @hlrichardson @rdvencioneck @JayalakshmiV @jayalakshmiV you are mentioned as maintainers of exos_ related stuff, so could you please take a look?

**Comment by plazukin** [comment]
> @plazukin hi, thanks for the PR!
> 
> * we should ensure this change is not breaking (for older versions of the underlying technology)
> * could you please add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) ?
> 
> cc @LindsayHill @ujwalkomarla @hlrichardson @rdvencioneck @JayalakshmiV @JayalakshmiV you are mentioned as maintainers of exos_ related stuff, so could you please take a look?

Changelog fragment added!
How can i help you with check the fix?

File: changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml
Comment (Andersson007) [inline changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml:3]:
```suggestion
  - exos_facts - returns timeout error when we use connection type ``network_cli``.
    It happened because we send command without ``no-refresh`` and script ``cli2json.py`` stuck in loop while reading console output (https://github.com/ansible-collections/community.network/pull/517).
```

**Review by Andersson007 — COMMENTED** [review]
@plazukin thanks for adding the fragment!
There's a small formatting thing to change.

As there are no integration tests, we cannot test it here.
Just be sure the `no-refresh` is supported in older versions of the service. Please check their docs to check.

Not being a network engineer, I provide only general support for the collection like releasing, CI, etc.
Let's wait for someones feedback until next week. If there's no feedback, I'll merge the PR after Fosdem, i.e. at the end of the next week.

Thank you

File: changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml
Comment (plazukin) [reply changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml]: I fixed it. Thank you

Comment (Andersson007) [reply changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml]: cool, thanks, so let's wait until the mid/end of the next week, then I'll merge if no objections

Comment (plazukin) [reply changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml]: Ok, i hope that all fine. 
Thank you :)

**Comment by plazukin** [comment]
@Andersson007, hi! 
It's time to merge it!
Can we do this? :)

**Review by Andersson007 — APPROVED** [review]

**Comment by Andersson007** [comment]
@plazukin hi, thanks for the reminder! busy time. Please do it in future PRs:)
Feel free also to add yourself as the module's maintainer to `.github/BOTMETA.yml` for the file or group of files.
The bot doesn't work at the moment but in future you'll be notified if someone reports, say, a bug:)
Also we'd be happy to see more contributions from you here and across other collections. I myself a maintainer of community.mysql and community.postrgresql (and i can help with others).
Here's also the [contributor path](https://docs.ansible.com/ansible/devel/community/contributor_path.html) which can guide you.
Any questions? Feel free to ask (join [Matrix channels](https://docs.ansible.com/ansible/devel/community/communication.html), at lease `community` and `social`).
Thanks for the contribution!

**Comment by Andersson007** [comment]
@plazukin could you please manually [backport](https://docs.ansible.com/ansible/devel/community/development_process.html#backporting-merged-prs-in-ansible-core) the changes to `stable-4`? It cannot be automatically backported by our bot, i guess, because the file lived in another path.

**Comment by Andersson007** [comment]
@plazukin could you please take a look ^?

**Comment by plazukin** [comment]
> @plazukin could you please take a look ^?

@Andersson007, sorry i was on vacation
Found new commits from the community. What do I need to do now? Do I need to make the backport `stable-4`?

**Comment by Andersson007** [comment]
> > @plazukin could you please take a look ^?
> 
> @Andersson007, sorry i was on vacation Found new commits from the community. What do I need to do now? Do I need to make the backport `stable-4`?

@plazukin yep, you can use https://docs.ansible.com/ansible/devel/community/development_process.html#backporting-merged-prs-in-ansible-core (use `main` instead of `devel` as the manual is for ansible core)

**Pull request merged**
